coderedcms/project_template/basic/project_name/settings/base.py

```diff
@@ -203,3 +203,7 @@
 # Tags
 
 TAGGIT_CASE_INSENSITIVE = True
+
+# Sets default for primary key IDs - change to BigAutoField if you prefer a 64-bit integar instead
+# See https://docs.djangoproject.com/en/3.2/ref/models/fields/#autofield
```

Comment:
Since these python files are technically templates, the django version in the URL (3.2) should be replaced with the template variable (see other urls in this file)

Reply:
done